js/logic.js

```diff
@@ -0,0 +1,177 @@
+  popupLinks.appendChild(liveLink);
+
+  popupBody2.appendChild(popupLinks);
+
+  popupBody.appendChild(popupBody2);
+
+  popupMain.appendChild(popupBody);
+
+  projectPopup.appendChild(popupMain);
+
+  popup.appendChild(projectPopup);
+  document.body.appendChild(popup);
+}
+
+window.onload = createPopup();
+
+const popupWrapper = document.querySelector('.project-popup-wrapper');
+const popupName = document.querySelector('.popup-name');
+const popupPosition = document.querySelector('.popup-position');
+const popupWorkName = document.querySelector('.popup-work-name');
+const popupImage = document.querySelector('.popup-image');
+const popupDescription = document.querySelector('.popup-description');
+const popupTech = document.querySelector('#popup-technologies-container');
+const projectOne = document.querySelector('#project-one');
+const projectTwo = document.querySelector('#project-two');
+const projectThree = document.querySelector('#project-three');
+const projectFour = document.querySelector('#project-four');
+const closePopup = document.querySelector('.close-project-popup');
+const blur = document.querySelector('#element');
+
+const popupOne = {
+  name: 'Tonic',
+  workName: 'CANOPY',
+  position: ['Back End Dev', '2015'],
+  imageUrl: './img/Snapshoot Portfolio.svg',
+  description:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
+  technologies: ['html', 'css', 'javascript'],
+};
+
+const popupTwo = {
+  name: 'Multi-Post Stories',
+  workName: 'FACEBOOK',
+  position: ['Full Stack Dev', '2015'],
+  imageUrl: './img/Snapshoot Portfolio2.svg',
+  description:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
+  technologies: ['html', 'css', 'javascript'],
+};
+
+const popupThree = {
+  name: 'Facebook 360',
+  workName: 'FACEBOOK',
+  position: ['Full Stack Dev', '2015'],
+  imageUrl: './img/Snapshoot Portfolio3.svg',
+  description:
```

Comment:
- [ ] I must commend your approach of displaying the project data dynamically just as the previous reviewer suggested 👍🏽 . I think it will be better if you add all your project data in a single array so that it will be easier to loop through the project data instead of repeating yourself 😄 

e.g

```
const projectDatas = [
   {
  name: 'Multi-Post Stories',
  workName: 'FACEBOOK',
  position: ['Full Stack Dev', '2015'],
  imageUrl: './img/Snapshoot Portfolio2.svg',
  description:
    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
  technologies: ['html', 'css', 'javascript'],
},
{
  name: 'Multi-Post Stories',
  workName: 'FACEBOOK',
  position: ['Full Stack Dev', '2015'],
  imageUrl: './img/Snapshoot Portfolio2.svg',
  description:
    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
  technologies: ['html', 'css', 'javascript'],
},
{
  name: 'Multi-Post Stories',
  workName: 'FACEBOOK',
  position: ['Full Stack Dev', '2015'],
  imageUrl: './img/Snapshoot Portfolio2.svg',
  description:
    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
  technologies: ['html', 'css', 'javascript'],
};
]
```